### DISCUSSION TITLE
Time-Varying MODIS Land Use Data

### DISCUSSION THREAD

**Discussion by osbornesci** [NASA-LIS/LISF#1143] [Q&A] [answered]
Greetings,
 
Our group is interested in investigating the sensitivity of LDT and LIS output to time-varying MODIS land use values (and comparing MODIS land use values to local observations). 
 
Currently, within the ldt.config, we are using the Discover igbp.bin binary file for MODIS land use. Does this igbp.bin file contain information for a particular temporal range and/or temporal resolution? 
 
We wonder, do there exist valid time-varying MODIS land cover files on Discover or elsewhere that can be used? If so, we’d be interested in modifying the ldt.config to direct to these files.
 
We would appreciate any suggestions.
Thank you kindly.

**Comment by karsenau** [comment, marked as answer]
Hi @osbornesci:

The igbp.bin file should be reflective of year 2001 MODIS landcover IGBP classes, with a few added by NCEP.  More details on that can be found here:
https://ral.ucar.edu/solutions/products/noah-multiparameterization-land-surface-model-noah-mp-lsm

Yes, MODIS landcover files that are available on the NASA DAACs are available for each year.  Current code in LIS master does not have the capability to support the individual years, e.g., change each year automatically.  Note:  Updates exist but currently in the works by colleagues in the community.

**Reply by osbornesci** [reply]
Hi @karsenau:

Thank you kindly for the helpful information!

We are now also in contact with Sujay regarding time-varying MCD12Q-based land cover files.

Thanks again.

**Reply by karsenau** [reply]
You're welcome.  

If you feel your questions have been answered, feel to select below the green "Answer" button. Thanks!

**Reply by osbornesci** [reply]
Will do! Thank you.

**Reply by alexzhang-psu** [reply]
Hello. I'm wondering if you have figured out how to apply time-varing MODIS land-use data in LIS? Thanks very much.

**Reply by zaitchik** [reply]
Hi Alex,

There might be smoother ways to do this, but for our team (I'm the PI on the group that Tashiana Osborne was a member of, and we initiated this thread) we've found it simplest to run LDT separately for each year and just input a different land cover map each time. That requires that we restart our LIS simulations each year to read in a new lis_input file, but it's pretty simple to script up.

Hope that helps!

Ben

**Reply by alexzhang-psu** [reply]
Thank you so much for answering this, Ben! I'm now also trying to read MCD12Q1 landcover data for each year's simulation, but there's some error message that I need to work on. I've read some papers from your group regarding incorporating irrigation info. in LIS which are great works (I'd like to follow what you've done and apply those in irrigation areas in California). Thanks again.

**Reply by zaitchik** [reply]
Great. Please reach out if you have trouble getting around those error messages. Also happy to talk about the irrigation work when you get to that step.
